stable/atlantis/README.md

```diff
@@ -46,8 +45,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | `podTemplate.annotations`                   | Additional annotations to use for the StatefulSet.                                                                                                                                                                                                                                                        | n/a     |
 | `logLevel`                                  | Level to use for logging. Either debug, info, warn, or error.                                                                                                                                                                                                                                             | n/a     |
 | `orgWhiteList`                              | Whitelist of repositories from which Atlantis will accept webhooks. **This value must be set for Atlantis to function correctly.** Accepts wildcard characters (`*`). Multiple values may be comma-separated.                                                                                             | none    |
-| `requireApproval`                           | Whether to require pull request approval prior to applies. See [Approved Requirement](https://www.runatlantis.io/docs/apply-requirements.html#approved).                                                                                                                                                  | `false` |
-| `requireMergeable`                          | Whether to require pull request to be mergeable prior to applies. See [Mergeable Requirement](https://www.runatlantis.io/docs/apply-requirements.html#mergeable).                                                                                                                                         | `false` |
+| `repoConfig`                                | Server Side Configuration in YAML, configuration is stored in ConfigMap                                                                                                                                                                                                                                   | n/a     |
```

Comment:
```suggestion
| `repoConfig`                                | [Server Side Repo Configuration](https://www.runatlantis.io/docs/server-side-repo-config.html) as a raw YAML string. Configuration is stored in ConfigMap.                                                                                                                                                                                                                                   | n/a     |
```

Just want to stress that this is repo configuration which is different from the normal server config. Also make it clear that this should be a raw yaml string.